pycalc.py

```diff
@@ -1,10 +1,8 @@
 def menu():
+    operations = ["Exit", "Sum", "Subtraction", "Multiplication", "Division"]
     print("### Menu ###")
-    print("[0] - Exit")
-    print("[1] - Sum")
-    print("[2] - Subtraction")
-    print("[3] - Multiplication")
-    print("[4] - Division")
+    for operation in operations:
+        print(f"[{ operations.index(operation) }] - { operation }")
```

Comment:
I think using `enumerate` would be a better choice here. Refer to the following example:

```
>>> for idx, n in enumerate(["a", "b", "c"]):
...   print(f"{idx}: {n}")
... 
0: a
1: b
2: c
```

This way you don't need to look up again using the `index` function.